packages/cozy-client/src/CozyClient.js

```diff
@@ -309,21 +310,26 @@ class CozyClient {
    *
    * Warning: unlike other instantiators, this one needs to be awaited.
    *
-   * @param {OldCozyClient} oldClient - An instance of the deprecated cozy-client
+   * @param {OldCozyClient} oldClient - An OAuth instance of the deprecated cozy-client
+   * @param {object} options - CozyStackClient options
    * @returns {Promise<CozyClient>} An instance of a client, configured from the old client
    */
   static async fromOldOAuthClient(oldClient, options) {
-    const hasOauthCreds = oldClient._oauth && oldClient._authcreds != null
```

Comment:
There is still a reference to `_authcreds` in `CozyClient.spec.js` in `can be instantiated from an oauth-based old client` test.

As I don't see any other ref of it, maybe we should remove this prop from the test?

Reply:
It's already been removed as part of a force push ;-)